build.sbt

```diff
@@ -68,7 +68,21 @@ libraryDependencies ++= Seq(
   "org.specs2" % "specs2_2.11" % "2.3.13" % "test"
 )
 
-scalacOptions ++= Seq("-unchecked", "-deprecation")
+// The OpenIE codebase generates compiler warnings like this:
+//
+//   Adapting argument list by creating a 2-tuple
+//   Adapting argument list by creating a 3-tuple
+//   implicit conversion method ... should be enabled
+//   method deserialize in object ... is deprecated
+//   possible missing interpolator
+//   reflective access of structural type member method close should be enabled
+//   side-effecting nullary methods are discouraged
+//   type parameter String defined in method apply shadows type String defined in object Predef. You may want to rename your type parameter, or possibly remove it.
+//
+// The code should be fixed to not have these problems. Until then, "-nowarn"
```

Comment:
I think that's fine. It won't bother users of the packaged library, and it might be annoying enough to push whoever ends up maintaining the library towards gradually fixing the problems.